en/docs/reference/faq.md

```diff
@@ -152,7 +142,14 @@ See [Working with Features](https://docs.wso2.com/display/ADMIN44x/Working+with+
 
 ### How can I preserve the CDATA element tag in API responses?
 
-Set the `javax.xml.stream.isCoalescing` property to `false` in the `<API-M_HOME>/XMLInputFactory.properties` file. Here's an example:
+Set the `javax.xml.stream.isCoalescing` property to `false` by adding the following configuration in `deployment.toml` file, which is in the `<API-M-Home>/repository/conf` directory. 
+
+``` toml
+[xml_input_factory.properties]
+"javax.xml.stream.isCoalescing" = false
+```
+
+Example:
 
 ``` java
```

Comment:
Is this example still valid?

Reply:
Yes, it is valid. It is a sample response from an API. The configuration needs to be added to preserve the `CDATA` element in the response.